test/e2e/tests/confirmations/signatures/signature-helpers.ts

```diff
@@ -99,3 +113,13 @@ export async function assertPastedAddress(driver: Driver) {
   const formFieldEl = await driver.findElement('#eip747ContractAddress');
   assert.equal(await formFieldEl.getAttribute('value'), WALLET_ADDRESS);
 }
+
+export async function openDappAndTriggerSignature(
```

Comment:
This is great and absolutely the best mentality!

But this is another example where the page object model (once adopted) would make this architecture even cleaner and more consistent.

```
await TestDApp.clickSignTypedData3Button();
await SignatureConfirmation.clickApprove();
```

Reply:
another idea is we could make this method even more generic. Possibly we could create test dapp helpers. 

e.g. calling it from the code
```
openDappAndClick(driver, Signature.SignTypedData)
```
or 
```
openDappAndClick(driver, Dapp.Signature.SignTypedData)
```